### ISSUE TITLE
Belt Obstacle Does Support Line Graphic in B Standard

### ISSUE THREAD

**Issue by david-melendez** [missioncommand/old-mil-sym-java#62]
According to both the MIL-STD-2525B and the MIL-STD-2525C, the **TACTICAL GRAPHICS->MOBILITY/SURVIVABILITY->OBSTACLES->GENERAL->BELT** obstacle is an area graphic requiring a minimum of three points to render the graphic.

However, in the USAS document for Standard B, the BELT obstacle has the option of being a linear graphic with two points (see A**CCS-A3-500-007A Rev 3, Updated 29 August 2016** as well as **ACCS-A3-500-007A Rev 4,  Updated 15 November 2016**: Unified Specification for ABCS Symbology (USAS) for the COP Quarterly Release to BC Marshall / Capability Set 11-12).

However, whenever I attempt to render this graphic with two points, as below, I get an error stating that the graphic requires at least three points to render the graphic.

```
String kml = r.RenderMultiPointSymbol("ID", "BELT",
                "BELT", "GFMPOGB---****X", 
                "-86.5870132446289,30.6727600097656 -86.5819625854492,30.6749858856201",
                "clampToGround", 50000, "", 
                "",
                0, 0);
```

Is there a way to generate a two-point BELT graphic in the renderer?

**Comment by mdeutch3** [comment]
The renderer does not currently support a 2 point Belt.

**Comment by mdeutch3** [comment]
The two point belt has been implemented. The source changes have been checked in.

**Comment by michael-spinelli** [comment]
will be fixed in release 0.1.22
